portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,112 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-const ham = document.querySelector('.ham')
+//var myBody = getElementByTagName('body');
 
-ham.addEventlisteber('click', function(){
-    this.classList.toggle('is-active');
-})
\ No newline at end of file
+
+// translating text in testing phase
+/*
+function requestTranslation() {
+    const text = document.getElementById('text').value;
+    const languageCode = document.getElementById('language').value;
+
+    const resultContainer = document.getElementById('result');
+    resultContainer.innerText = 'Loading...';
+
+    const params = new URLSearchParams();
+    params.append('text', text);
+    params.append('languageCode', languageCode);
+
+    fetch('/translate', {
+      method: 'POST',
+      body: params
+    }).then(response => response.text())
+    .then((translatedMessage) => {
+      resultContainer.innerText = translatedMessage;
+    });
+  }
+*/
+
+// time bottom for fetch
+async function showServerTime(event) {
+    event.preventDefault();
+
+    const data = new FormData(event.target);
+    const value = Object.fromEntries(data.entries());
+
+
+    const responseFromServer = await fetch('/hello' , {
+        method: 'POST', // or 'PUT'
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(value),
+      });
+    const textFromResponse = await responseFromServer.text();
+  
+    const helloPara = document.getElementById('name-container');
+    helloPara.innerText = `Hello ${textFromResponse} `;
+}
+// document.getElementById('form2').addEventListener('submit', showServerTime);
+
+
+// const ham = document.querySelector('.ham')
+
+// ham.addEventListener('click', function(){
+//    this.classList.toggle('is-active');
+// })
+
+window.onload = function(){
+    document.getElementById('form2').addEventListener('submit', showServerTime);
+
+
+    const ham = document.querySelector('.ham')
+
+    ham.addEventListener('click', function(){
+        this.classList.toggle('is-active');
+    })
+  };
+
+
+
+  //showing messages for step3 of week2
+  /** Fetches stats from the server and adds them to the page. */
+async function getRandomMessage() {
+    const responseFromServer = await fetch('/message');
```

Comment:
"/message" does not correspond to any servlet, did you implement/added it?

Reply:
Actually it correspond to the MessageServlet.java file